python/dolfinx/jit.py

```diff
@@ -90,35 +90,40 @@ def mpi_jit(*args, **kwargs):
 
 
 @mpi_jit_decorator
-def ffcx_jit(ufl_object, form_compiler_parameters=None):
+def ffcx_jit(ufl_object, form_compiler_parameters={}, jit_parameters={}):
     # Prepare form compiler parameters with overrides from dolfinx
     p = ffcx.default_parameters()
     p["scalar_type"] = "double complex" if common.has_petsc_complex else "double"
-    p.update(form_compiler_parameters or {})
+    p.update(form_compiler_parameters)
 
     # CFFI compiler options/flags
-    extra_compile_args = ['-g0', '-O3', '-march=native']
-    user_cflags = os.getenv('DOLFINX_JIT_CFLAGS')
-    if user_cflags is not None:
-        extra_compile_args = user_cflags.split(" ")
-    cffi_options = dict(cffi_extra_compile_args=extra_compile_args, cffi_verbose=False,
-                        cffi_debug=False)
+    jit_params = {"cffi_extra_compile_args": ["-O3", "-g0", "-march=native"],
```

Comment:
I would prefer less aggressive defaults  - aggressive optimisation can add a lot to the compile time.

Reply:
I thought we agreed to get rid of `-march=native`

Reply:
I just copied existing default args in this PR. Didn't want to touch defaults, but I can do more at once...